contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -139,6 +132,15 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5, DaoOperator
             rewardInBps: _defaultRewardBps,
             rebateInBps: _defaultRebateBps
         });
+
+        // deploy staking contract 
+        staking = new KyberStaking({
+            _kncToken: _knc,
+            _epochPeriod: _epochPeriod,
+            _startTimestamp: _startTimestamp,
+            _daoContract: address(this)
+        });
+        emit KyberStakingDeployed(address(staking));
```

Comment:
Don't think an event is required here.
anyway the address can be easily fetched. 
And will never have another one

Reply:
ok, can remove event, staking is public getter anyway